src/pudl/convert/metadata_to_rst.py

```diff
@@ -51,7 +52,8 @@ def main():
     # Sort fields within each resource by name:
     for resource in package.resources:
         resource.schema.fields = sorted(resource.schema.fields, key=lambda x: x.name)
-    package.to_rst(path=args.output)
+    docs_dir = Path(__file__).parent[3].resolve() / "docs"
```

Comment:
This maybe isn't the best way to get the docs directory. It's now needed because the `package.rst.jinja` template now lives in the templates in the docs directory.

Reply:
This will fail any time the code is being run after being installed, since it gets installed in a directory that isn't the repository, and only the repository is assured to have the `docs` directory in that location relative to the source code. Can we pass the path to the documentation directory in as an argument instead, and give it to the `metadata_to_rst` script on the command line when it's called?

Reply:
Ah yep this was a placeholder that I forgot to change. Updated so the `docs` directory is passed as an argument.